js/classes/game_engine.js

```diff
@@ -132,92 +170,32 @@ class GameEngine {
      * This method is going to start listening for user inputs. Affects the Game Engine's left/right/up/down booleans according
      * to interaction with either the WASD keys or arrows.
      */
-    startInput() {
-
-        CANVAS.addEventListener("mousedown", (e) => {
-            if (e.button === 0) {
-                this.user.firing = false;
-                this.user.aiming = true;
-            } else if (e.button === 2) {
-                // TODO update user.jabbing here
-            }
-        });
-
-        CANVAS.addEventListener("mouseup", (e) => {
-            if (e.button === 0) {
-                this.user.aiming = false;
-                this.user.firing = true;
-            } else if (e.button === 2) {
-                // TODO update user.jabbing here
-            }
+    configureEventListeners() {
```

Comment:
I like this method name much better than the old one.

Reply:
Me too. startInput() was from prof's code lol. I never liked it.